app/src/containers/App.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
Remove this if it's not needed. Test file can stay in your local. Only push the code that will be used on production.